api/users/pagination.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+from django.core.urlresolvers import reverse
+
+from api.base.pagination import JSONAPIPagination
+from rest_framework.response import Response
+
+class UserNodeLogPagination(JSONAPIPagination):
+
+    node_log_aggregates = None
```

Comment:
How does setting this attribute allow you to call get_aggregates? How does `node_log_aggregates` map back to `get_aggregates`?